src/sentry/integrations/pipeline.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import absolute_import, print_function
+
+__all__ = ['IntegrationPipeline']
+
+import json
```

Comment:
Use `sentry.utils.json` instead unless there was a reason not to.

Reply:
[Updated here](https://github.com/getsentry/sentry/pull/6708/files#diff-94419ddbfb31de69ee4e9fefa0cd2a87R11).